plugin/web/extensions/sidebar/core/components/hooks/projectHooks.ts

```diff
@@ -279,9 +279,10 @@ export default ({
               handleProjectDatasetAdd(mergedDataset);
             }
           });
-          if (data.userStory.length > 0) {
+          if (data?.userStory?.length > 0) {
```

Comment:
You'll need to do `if(data.userStory && data.userStory.length > 0_)` We check if (data) above already